### DISCUSSION TITLE
手势里的贴图如何设置到全局热键里面啊？

### DISCUSSION THREAD

**Discussion by cc19264997** [shuax/MouseInc.Settings#284] [Q&A]
点都搞唔明。。。。

**Comment by cc19264997** [comment]
复制了贴图里的Raw到全局热键，然后会选择最前方的面板自动截图，不能框选，这可怎么办

**Comment by shuax** [comment]
```
[
	["Snapshot", "Stick"]
]
```

**Comment by cc19264997** [comment]
QAQ，，，还有音量加大调小的全局热键的代码我在官网找不到。。

**Comment by cc19264997** [comment]
牛逼！再请教下如何全局让它截图的同时复制到剪切板




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "shuax/MouseInc.Settings"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年6月7日(星期二) 上午10:46
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [shuax/MouseInc.Settings] 手势里的贴图如何设置到全局热键里面啊？ (Discussion #284)





 [ 	["Snapshot", "Stick"] ]
